code/_globalvars/lists/names.dm

```diff
@@ -10,8 +10,6 @@ GLOBAL_LIST_INIT(last_names, file2list("config/names/last.txt"))
 GLOBAL_LIST_INIT(clown_names, file2list("config/names/clown.txt"))
 GLOBAL_LIST_INIT(mime_names, file2list("config/names/mime.txt"))
 GLOBAL_LIST_INIT(golem_names, file2list("config/names/golem.txt"))
-GLOBAL_LIST_INIT(megacarp_first_names, file2list("strings/names/megacarp1.txt"))
-GLOBAL_LIST_INIT(megacarp_last_names, file2list("strings/names/megacarp2.txt"))
```

Comment:
either all of these configs are global or none of them are. What the heck is this

Reply:
All of the rest of them are used in multiple files (Except the ninja names apparently), whereas the megacarp names are used in a single file for a single rare mob. I figured that it was worth the inconsistency in this case.

Reply:
it is better to keep consistancy then to deviate off and make it inconsistent in best practice.

Reply:
I've thought about it a bit more, and yeah it would be better to keep it as it is.